Sources/Datadog/Core/DD/TracingHeaderType.swift

```diff
@@ -0,0 +1,13 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+/// The type of the telemetry tracing header injected to requests.
+public enum TracingHeaderType {
```

Comment:
I think we should move this to [`FeaturesConfiguration .Tracing `](https://github.com/DataDog/dd-sdk-ios/blob/develop/Sources/Datadog/Core/FeaturesConfiguration.swift#L43), so all configuration is defined there.

Reply:
I ended up placing it here, because this type was used in a bunch of different modules. Made sense to me to move it up to core.

I will give it another look for sure.

Reply:
It's actually already a part of `FeaturesConfiguration.URLSessionAutoInstrumentation`, but since the model is used in the public SDK configuration I had to place elsewhere to make it public.

Reply:
Ok, so maybe we can move it to `Sources/Datadog/URLSessionAutoInstrumentation`?
As we migrate toward v2, we will move all definitions to their respective Features.